source/seismology/software.rst

```diff
@@ -1,2 +1,103 @@
 常用软件
 ========
+
+:本节贡献者: |姚家园|\（作者）、
+             |田冬冬|\（审稿）
+:最近更新日期: 2021-05-30
+:预计阅读时间: 30 分钟
+
+----
+
+日常地震学科研工作中，经常需要使用软件来完成数据处理与分析、绘图以及科学计算等。
+除非特殊情况，大部分我们可以直接使用他人开发好的软件。本节介绍一些地震学常用软件，
+读者需要在后面的练习以及日常的科研工作中，反复使用和思考方能完全掌握这些软件。
+
+.. note::
+
+   阅读本教程时，推荐地震学新手先参考以下中文教程中的安装部分在计算机上安装这些软件。
+   《\ :doc:`地震学实践 </exercises/introduction>`\ 》这一章则会通过简单实例展示一些基本用法。
+
+   阅读完本教程后，再详细阅读各软件的中文和官方教程。
+   一般而言，只需阅读中文教程中的入门教程，即可入门和开展科研工作。
+   之后可以阅读中文教程中的进阶教程以及官方教程，深入学习和掌握这些软件。
+
+ObsPy
+------
+
+ObsPy 是地震学数据处理的 Python 软件包，极大促进了地震学应用程序的快速开发。
+其可以用来读写多种地震学数据格式，下载地震目录、台站元数据和波形数据，以及处理和分析波形数据等，
+满足了日常科研中与地震学数据相关的大部分需求。
+
+- 主页：https://www.obspy.org/
+- 官方教程：https://docs.obspy.org/
+- 中文教程：https://seismo-learn.org/software/obspy/
+
+可以说正是因为 ObsPy 的出世，地震学科研工作者才可以喊出“\ **人生苦短，我用 Python！**\ ”。
+
+SOD
+---
+
+SOD，全称是 Standing Order for Data，可以自动筛选并下载地震目录、台站元数据和波形数据，
+并对波形数据做预处理。其具有高度可定制化的特点，可以满足日常科研中地震学数据下载的大部分需求。
+
+- 主页：http://www.seis.sc.edu/sod/
+- 中文教程：https://seismo-learn.org/software/sod/
+
+.. note::
+
+   ObsPy 和 SOD 都可以用于下载数据。这里之所以推荐读者使用 SOD 主要有两方面原因：
```

Comment:
We can list SOD here, but better not recommend it.

Reply:
done in f8ef8a6.

Reply:
But we don't maintain SOD recipes on https://seismo-learn.org/software/sod/ yet. Shall we add a quick link to SOD tutorials?

Reply:
We can migrate the SOD tutorial quickly if necessary.

Reply:
We may make SOD the highest priority after this PR.